src/language-server/__tests__/fixtures/documents/commentWithTemplate.ts.snap

```diff
@@ -0,0 +1,185 @@
+>// hint for reading the snapshots generated by this file:
+#^^ source.ts comment.line.double-slash.ts punctuation.definition.comment.ts
+#  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ts comment.line.double-slash.ts
+>// it is important to check that a rule doesn't only enter `meta.embedded.block.graphql`
+#^^ source.ts comment.line.double-slash.ts punctuation.definition.comment.ts
+#  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ts comment.line.double-slash.ts
+>// but also leaves it again
+#^^ source.ts comment.line.double-slash.ts punctuation.definition.comment.ts
+#  ^^^^^^^^^^^^^^^^^^^^^^^^^ source.ts comment.line.double-slash.ts
+>
+>/* NormalComment */ `query Foo {test}`;
+#^^ source.ts comment.block.ts punctuation.definition.comment.ts
+#  ^^^^^^^^^^^^^^^ source.ts comment.block.ts
+#                 ^^ source.ts comment.block.ts punctuation.definition.comment.ts
+#                   ^ source.ts
+#                    ^ source.ts string.template.ts punctuation.definition.string.template.begin.ts
+#                     ^^^^^^^^^^^^^^^^ source.ts string.template.ts
+#                                     ^ source.ts string.template.ts punctuation.definition.string.template.end.ts
+#                                      ^ source.ts punctuation.terminator.statement.ts
+>
+>// prettier-ignore
+#^^ source.ts comment.line.double-slash.ts punctuation.definition.comment.ts
+#  ^^^^^^^^^^^^^^^^ source.ts comment.line.double-slash.ts
+>{
+#^ source.ts meta.block.ts punctuation.definition.block.ts
+>const _1 = /* GraphQL */`query Q1 {test}`;
+#^^^^^ source.ts meta.block.ts meta.var.expr.ts storage.type.ts
+#     ^ source.ts meta.block.ts meta.var.expr.ts
+#      ^^ source.ts meta.block.ts meta.var.expr.ts meta.var-single-variable.expr.ts meta.definition.variable.ts variable.other.constant.ts
+#        ^ source.ts meta.block.ts meta.var.expr.ts meta.var-single-variable.expr.ts
+#         ^ source.ts meta.block.ts meta.var.expr.ts keyword.operator.assignment.ts
+#          ^ source.ts meta.block.ts meta.var.expr.ts
+#           ^^^ source.ts meta.block.ts meta.var.expr.ts
+#              ^^^^^^^ source.ts meta.block.ts meta.var.expr.ts markup.italic
```

Comment:
I've chosen to make the word `GraphQL` in the `/* GraphQL */` comment here italic to highlight that it does something.